### ISSUE TITLE
Error in irlba::irlba(L, nv = n, nu = 0, maxit = iters) :    function 'sexp_as_cholmod_sparse' not provided by package 'Matrix'

### ISSUE THREAD

**Issue by yulchen810** [satijalab/seurat#8242]
Hi,
I already downgrade Matrix to version 1.6.1.1 and reinstalled irlba, but this error still comes up when I run FindNeighbors. does anyone  encounter this problem? 
thank you very much

**Comment by yulchen810** [comment]
> Hi, I already downgrade Matrix to version 1.6.1.1 and reinstalled irlba, but this error still comes up when I run FindNeighbors. does anyone encounter this problem? thank you very much

this error has been resolved by downgrade Matrix package to 1.5-3

**Comment by Gesmira** [comment]
Hi @yulchen810, 
I also recommend the following: 
```
install.packages("Matrix", type = "source")
install.packages("irlba", type = "source")
```
It seems to have resolved similar issues: https://github.com/satijalab/seurat/issues/8100

**Comment by yulchen810** [comment]
> Hi @yulchen810, 
> I also recommend the following: 
> ```
> install.packages("Matrix", type = "source")
> install.packages("irlba", type = "source")
> ```
> It seems to have resolved similar issues: https://github.com/satijalab/seurat/issues/8100
>  

I also tried this, but it didn't work for me.
Best

**Comment by davhg96** [comment]
Same issue for me but when running the RunVSD function for ATAC seq analysis

**Comment by martin284** [comment]
> > Hi, I already downgrade Matrix to version 1.6.1.1 and reinstalled irlba, but this error still comes up when I run FindNeighbors. does anyone encounter this problem? thank you very much
> 
> this error has been resolved by downgrade Matrix package to 1.5-3

Hi @yulchen810 
What versions of R and Seurat did you use to get it to work? I have the same problem and using Matrix version 1.5-3 leads to new conflicts as my Seurat version requires a newer Matrix version.

**Comment by yulchen810** [comment]
> > > Hi, I already downgrade Matrix to version 1.6.1.1 and reinstalled irlba, but this error still comes up when I run FindNeighbors. does anyone encounter this problem? thank you very much
> > 
> > this error has been resolved by downgrade Matrix package to 1.5-3
> 
> Hi @yulchen810 
> What versions of R and Seurat did you use to get it to work? I have the same problem and using Matrix version 1.5-3 leads to new conflicts as my Seurat version requires a newer Matrix version. 

R: 4.2.3
Seurat: 4.3.0.1

**Comment by martin284** [comment]
Thanks! Only as an addition:

R: 4.3.2
Seurat: 4.4.0
Matrix: 1.6-1.1

also works for me.

**Comment by WangGuixiangCoder** [comment]
Same issue exists. when R version is 4.3.3  and Seurat is 5.0.1, Matrix is 1.6_4 works for me.

`conda install conda-forge::r-matrix=1.6_4`



> > Hi @yulchen810,
> > I also recommend the following:
> > ```
> > install.packages("Matrix", type = "source")
> > install.packages("irlba", type = "source")
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > It seems to have resolved similar issues: #8100
> 
> I also tried this, but it didn't work for me. Best
